app/models/comment.rb

```diff
@@ -34,4 +34,12 @@ def self.populate_comment(comment_params, question, answer)
       question.comments.create(body: comment_params[:body])
     end
   end
+
+  def self.check_comments
```

Comment:
Sweet method for checking comments on an object. This should be able to translate over to use for answers also if the same thing breaks.